[Bug]: Bootstrap.Common.cs 编译报错：SceneHandle 存在引用歧义

### JEngine Version

e.g 1.0.9

### Unity Version

e.g Unity6000.5

### Target Platform

Windows

### Bug Description

## 错误描述 (Description)
在使用 JEngine 时，项目出现编译错误。报错指向 `Bootstrap.Common.cs` 文件，提示 `SceneHandle` 在 `UnityEngine.SceneManagement.SceneHandle` 和 `YooAsset.SceneHandle` 之间存在二义性引用。

这似乎是因为项目中同时引入了 `UnityEngine.SceneManagement` 和 `YooAsset` 命名空间，而两者都包含 `SceneHandle` 类型，导致编译器无法确定使用哪一个。

## 报错信息 (Error Log)
```text
Packages\com.jasonxudeveloper.jengine.core\Runtime\Bootstrap.Common.cs(45,37): error CS0104: 'SceneHandle' is an ambiguous reference between 'UnityEngine.SceneManagement.SceneHandle' and 'YooAsset.SceneHandle'

### Steps to Reproduce

在 Unity 项目中导入 JEngine 框架。


### Expected Behavior

代码应能正常编译，不应产生命名空间冲突。

### Error Logs

```shell

```

### Additional Context

_No response_

### Checklist

- [x] I have searched existing issues to ensure this bug hasn't been reported before
- [x] I have tested with the latest version of JEngine

--- a/UnityProject/Packages/com.jasonxudeveloper.jengine.core/Runtime/Bootstrap.Common.cs
+++ b/UnityProject/Packages/com.jasonxudeveloper.jengine.core/Runtime/Bootstrap.Common.cs
@@ -29,6 +29,7 @@ using JEngine.Core.Encrypt;
 using JEngine.Core.Update;
 using UnityEngine.SceneManagement;
 using YooAsset;
+using SceneHandle = YooAsset.SceneHandle;
 
 namespace JEngine.Core
 {

